ext/node/polyfills/internal/crypto/cipher.ts

```diff
@@ -342,6 +350,7 @@ export class Decipheriv extends Transform implements Cipher {
       this.#cache.cache,
       buf,
       this.#authTag || NO_TAG,
+      this.#autoPadding,
```

Comment:
The tests are failing because `buf` is a fixed buffer of size 16 (block size).

When auto padding is disabled, `buf` should be sliced from `(0, len]` where `len` is the length of the unpadded block returned by  `decrypt_block_b2b_mut`